### ISSUE TITLE
Misc thoughts and suggestions

### ISSUE THREAD

**Issue by LetterRip** [Rochester-NRT/RocAlphaGo#45]
I'm curious if rotations and reflections of the board would give the same value and policy outputs.  If they give different values would be interesting to explore if averaging them converges to the correct result faster.  I suspect human cognitive biases result in humans playing stronger for particular orientations than for others.

Another thought was predicting score for parts of the board and using those predictions to predict the overall value.  Again this seems like it might be a way to converge to the correct answer faster.  The same edge/corner relationships will occur in all four corners and four edges.

I noticed that the paper didn't seem to use CLOP for parameter tuning (not that they mention many parameters to tune).  It has given nice boosts in strength for other Go programs, might be worth looking at.

CLOP: Confident Local Optimization for Noisy Black-Box Parameter Tuning
http://www.remi-coulom.fr/CLOP/

http://www.mail-archive.com/computer-go@dvandva.org/msg04369.html

**Comment by wrongu** [comment]
Rotations/reflections are actually used by DeepMind's AlphaGo already! They do this during training to get more examples and during evaluation like this:

> we exploit symmetries at run-time by dynamically transforming each position s using the dihedral group of eight reflections and rotations, d1(s), …, d8(s). In an explicit symmetry ensemble, a mini-batch of all 8 positions is passed into the policy network or value network and computed in parallel.
(Silver et al. page 8)

Not sure what you mean by predicting scores for parts of the board - could you elaborate?

Optimizing hyperparameters parameters for a speed/accuracy tradeoff would not be a bad idea. Thanks for the link. Something to keep in mind once we get to a prototype.
